tools/deployment/package-helm/templates/reducer-deployment.yaml

```
@@ -0,0 +1,74 @@
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "clp.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: "reducer"
+  template:
+    metadata:
+      labels:
+        {{- include "clp.labels" . | nindent 8 }}
+        app.kubernetes.io/component: "reducer"
+    spec:
+      serviceAccountName: {{ include "clp.fullname" . }}-job-watcher
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)